inst/string.m

```diff
@@ -148,23 +149,30 @@
           error ('string: cell inputs must be cellstr');
         endif
         this.strs = in;
-        this.tfMissing = false (size (this.strs));
+        tfM = false (size (this.strs));
+        tfM(cellfun (@(x) isempty(x), this.strs)) = true;
+        this.tfMissing = tfM;
       elseif isnumeric (in)
         this.strs = arrayfun (@(x) {num2str(x)}, in);
-        this.tfMissing = false (size (this.strs));
+        tfM = false (size (this.strs));
+        tfM(arrayfun (@(x) isnan(x), in)) = true;
```

Comment:
I don't think you need `arrayfun` here, because `isnan` itself is already array-oriented/vectorized?

Reply:
That's true, we can just use `tfM(isnan (in) = true`.